tests/functional/modules/test_zos_find_func.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
```

Comment:
Tests look great, maybe i missed it but do you have a negative test case for both a non-existent data set and data set member?

Reply:
I currently do not! Thank for you for bringing that up and I will update the test cases accordingly.

Reply:
Thank you @asifmahmud